nin/dasBoot/lib/ConvolutionShader.js

```diff
@@ -83,7 +83,7 @@ THREE.ConvolutionShader = {
 
 		values = new Array( kernelSize );
 		sum = 0.0;
-		for ( i = 0; i < kernelSize; ++i ) {
+		for ( i = 0; i < kernelSize; ++ i ) {
```

Comment:
I wouldn't have made that change personally. But this is threejs linting rules ^^